# Controller settings
float32 p_in
float32 i_in
float32 d_in
float32 bias
uint8 control_mode